dynosaur_derive/src/lib.rs

```diff
@@ -154,7 +156,10 @@ pub fn dynosaur(
         }
     }
 
-    let vis = &item_trait.vis;
+    let vis = match attrs.vis {
+        Visibility::Public(_) | Visibility::Restricted(_) => &attrs.vis,
+        Visibility::Inherited => &item_trait.vis,
```

Comment:
This should just be nothing (private) if we're defaulting to that.

Reply:
lol, I was saying I agree with Niko and implemented this so quickly that I didn't realize I was doing the opposite thing.